package.json

```diff
@@ -272,8 +272,8 @@
 		"prebuild:packages": "npm run clean:packages && lerna run build",
 		"build:packages": "npm run --silent build:package-types && node ./bin/packages/build.js",
 		"build:plugin-zip": "bash ./bin/build-plugin-zip.sh",
-		"clean:package-types": "tsc --build --clean",
-		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
+		"clean:package-types": "tsc --build --clean; rimraf \"./packages/*/build-types\"",
```

Comment:
```suggestion
		"clean:package-types": "tsc --build --clean && rimraf \"./packages/*/build-types\"",
```

This format caused an error on Windows. Perhaps `&&` is better?

![image](https://github.com/WordPress/gutenberg/assets/54422211/d407bca1-8b1e-401f-9e6f-e492d435514b)

Reply:
Would it work on Windows if there were space before the `;`?

```suggestion
		"clean:package-types": "tsc --build --clean ; rimraf \"./packages/*/build-types\"",
```